modules/react/button/stories/visual-testing/stories_PrimaryButton.tsx

```diff
@@ -17,7 +17,12 @@ export const PrimaryButtonStates = (props: {theme?: PartialEmotionCanvasTheme})
   <StaticStates theme={props.theme}>
     <ComponentStatesTable
       rowProps={permutateProps({
```

Comment:
Can you [filter out](https://github.com/Workday/canvas-kit/blob/c11b43993e6e9a73555b6dcdbb74aa9025b489b1/modules/labs-react/common/lib/storybook-utils/permutateProps.ts#L5) some of the invalid combinations here as discussed?

Reply:
Got it sorted. 👍🏻